sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Azure.Data.SchemaRegistry.csproj

```diff
@@ -14,9 +14,15 @@
     <PackageReference Include="Azure.Core" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Temporary, until we move SchemaValidator and LruCache into core.-->
+    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Comment:
We'll have to release Experimental then change this to a package reference before releasing the Schema Registry beta.

Reply:
Tracking: https://github.com/Azure/azure-sdk-for-net/issues/37777